src/tools/piv-tool.c

```diff
@@ -494,10 +494,7 @@ int main(int argc, char * const argv[])
 			action_count++;
 			break;
 		case 'Z':
-			/*
-			-Z is the same as -C but -Z says the cert is compressed.
-			https://github.com/OpenSC/OpenSC/commit/3a92bf7af5aaa85fdbc0434d0a55393e68e0f98d#commitcomment-9636487
-			*/
+			/* fall through */
```

Comment:
I think this comment needs to go right above the next case (where usually the `break;` would be placed)